src/main/resources/db/migration/V1_97__create_draft_referral_update_referral_constraints.sql

```diff
@@ -0,0 +1,42 @@
+INSERT INTO metadata (table_name, column_name, sensitive, domain_data)
+  select 'draft_referral',column_name,sensitive,domain_data from metadata  where table_name='referral';
+
+CREATE TABLE draft_referral AS SELECT * FROM referral;
```

Comment:
During this transaction, old pods still using the `referral` table for drafts can attempt to insert; is that ok?

Reply:
- For the first point, probably we can mitigate that by copying the referrals where sentAt is null again to draft referrals manually